components/hab/src/main.rs

```diff
@@ -469,7 +469,12 @@ fn sub_pkg_binds(m: &ArgMatches) -> Result<()> {
 
 fn sub_pkg_dependencies(m: &ArgMatches) -> Result<()> {
     let ident = PackageIdent::from_str(m.value_of("PKG_IDENT").unwrap())?;
-    command::pkg::dependencies::start(&ident, &*FS_ROOT)
+    let scope = match m.is_present("TRANSITIVE") {
+        true => command::pkg::Scope::PackageAndDependencies,
+        false => command::pkg::Scope::Package,
+    };
```

Comment:
Why not
```rust
    let scope = if m.is_present("TRANSITIVE") {
        command::pkg::Scope::PackageAndDependencies
    } else {
        command::pkg::Scope::Package,
    };
```

Reply:
I go back and forth as well. Also about where to use `if let` vs `match`. My rule of thumb so far has been `match` whenever the exhaustiveness checking is likely to catch a logic error or will result in simpler code.